.github/actions/milestone-it/entrypoint.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+set -e
+
+# 1. Read current version.
+
+version=$(jq -r '.version' package.json)
+
+IFS='.' read -ra parts <<< "$version"
+major=${parts[0]}
+minor=${parts[1]}
+
+# 2. Determine next milestone.
+
+if [ minor == '9' ]; then
+	major=$((major+1))
+else
+	minor=$((minor+1))
+fi
+
+milestone="Gutenberg $major.$minor"
+
+# 3. Create milestone. This may fail for duplicates, which is expected and
+#    ignored.
+
+curl \
+	--silent \
+	-X POST \
+	-H "Authorization: token $GITHUB_TOKEN" \
+	-H "Content-Type: application/json" \
+	-d "{\"title\":\"$milestone\"}" \
```

Comment:
would be good to set the description and the date automatically as well (the date might be harder)

Reply:
> would be good to set the description and the date automatically as well (the date might be harder)

We don't currently have a description for these milestones. Maybe something like..

>Tasks to be included in the Gutenberg 5.8 plugin release.

?

Date... would be harder 😆 I'll think on how it might be implemented, but otherwise I could imagine it to be a future enhancement.

Reply:
> Tasks to be included in the Gutenberg 5.8 plugin release.

WFM

Reply:
> the date automatically as well (the date might be harder)

A thought on how this might be implemented:

1. Compute the number of versions which should have elapsed since we started strictly adhering to the two week release rhythm (or, any reference version to compare against which falls in this range).
2. Use `date` command with arithmetic to offset the start date (the date of the first bi-weekly release) by the `2 * numVersionElapsed`

e.g.

```
num_versions=8 # 5.8 - 5.0
weeks=$((num_versions*2))
reference_start_date=1549411200 # 5.0 release
due=$(date -u -r $reference_start_date -v+$(echo $weeks)w)
```

```
echo $due
# Wed May 29 00:00:00 UTC 2019
```

Unsure if this will be made complicated by differences in the `date` command between OS (macOS [BSD `date`](https://www.freebsd.org/cgi/man.cgi?date) vs. Ubuntu [GNU `date`](http://man7.org/linux/man-pages/man1/date.1.html), specifically on arithmetic via [`-v`](https://stackoverflow.com/questions/498358/how-do-i-do-date-math-in-a-bash-script-on-os-x-leopard) vs. [`-d`](https://unix.stackexchange.com/questions/49053/linux-add-x-days-to-date-and-get-new-virtual-date))